mk/Common.mk

```diff
@@ -183,8 +183,32 @@ endif
 # SVLIB repo var end
 
 ###############################################################################
-# Read YAML test specifications
+# Imperas Instruction Set Simulator
```

Comment:
I am going to leave this comment here, as the files in question are not changed by this PR - the makefile changes looks good, but it will also lead to some dead code in the various python scripts located in /bin, these will also need some cleanup if these changes are made here. (Not strictly necessary functionality-wise, but we would not want to bloat our codebase with dead code)

Reply:
Agreed, but this will not be resolved in this PR.   I will generate a seperate issue for this.